azure-pipelines.yml

```diff
@@ -44,6 +44,54 @@ variables:
   forwardCompatibleTestOptions_Windows_3_0_2: "--filter FullyQualifiedName=NONE"
   forwardCompatibleTestOptions_Linux_3_0_2: $(forwardCompatibleTestOptions_Windows_3_0_2)
 
+  # Skip backwardCompatible tests because Microsoft.Spark.Worker requires Spark 3.1 support in
```

Comment:
So it seems that we need 2.0?

Assuming we release 1.1, if the client is using 1.1 for Spark 3.1, and the worker installed is 1.0, things will break. Right?

Reply:
Yes, the 1.0.0 Worker will break any time we introduce a version that supports a new minor version of Spark.

Reply:
Quick question, why is the above ^ a must? If the Spark minor version doesn't involve changes to how stream is sent, can't we just do what Terry suggests [here](https://github.com/dotnet/spark/pull/836/files/91be8f4ceaadd06f1e2c403ee46cb307732f5bd5#r593972415)?

Reply:
No one said it's a must, it's just the current pattern we have it set up with.  The suggestion that Terry brought up was already discussed [here](https://github.com/dotnet/spark/pull/833#discussion_r589985702)